### DISCUSSION TITLE
[POSSIBLE BUG] Float list transformed to Float

### DISCUSSION THREAD

**Discussion by Axyom** [JacquesLucke/animation_nodes#1823] [Q&A]
Hi,

I've encountered a problem while trying to follow this tutorial : https://www.youtube.com/watch?v=GkG6OpWpDKQ

It seems that I follow all the steps, but my nodes keep creating a create node list when I try to connect a loop to other nodes.
Here are the screenshots depicting the situation :
![image](https://user-images.githubusercontent.com/32022099/150418682-b2760407-1f64-4d02-9a80-0036dae36af0.png)

![image](https://user-images.githubusercontent.com/32022099/150418727-f712915a-8f5d-4617-8e63-adc2d8a68967.png)

It seems that only the last element is taken into account.

I'm using Blender 3.0.0 and Animation Nodes 2.3.0

Here's the .blend : 
[Animation Nodes Tuto.zip](https://github.com/JacquesLucke/animation_nodes/files/7908386/Animation.Nodes.Tuto.zip)

**Comment by OmarEmaraDev** [comment]
I haven't checked the tutorial, but perhaps the float list was meant to be passed as a parameter not an iterator.
